tests/network/probes.go

```diff
@@ -106,21 +111,13 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 			if disableEnableCycle {
 				By("Disabling the guest-agent")
 				Expect(console.LoginToFedora(vmi)).To(Succeed())
-
-				Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
-					&expect.BSnd{S: "sudo systemctl stop qemu-guest-agent\n"},
-					&expect.BExp{R: console.PromptExpression},
-				}, 120)).ToNot(HaveOccurred())
+				Expect(stopGuestAgent(vmi)).ToNot(HaveOccurred())
```

Comment:
I think `To(Succeed())` fits better here.

Reply:
This was changed. Ignoring.